src/object/srv_obj.c

```diff
@@ -142,7 +141,7 @@ obj_rw_reply(crt_rpc_t *rpc, int status, uint64_t epoch,
 		if (rc != 0)
 			D_ERROR("send reply failed: "DF_RC"\n", DP_RC(rc));
 	} else {
-		D_WARN("lost reply rpc %p\n", rpc);
+		D_WARN("lost reply rpc %p opc %d\n", rpc, opc_get(rpc->cr_opc));
```

Comment:
There is a RPC_ERROR macro, but not RPC_WARN yet, it may be better to add a RPC_WARN macro and call it from here, as that will fill in the opcode/rank/tag automatically and in a pre-defined format.

Reply:
good idea...